lib/debt_management_center/financial_status_report_service.rb

```diff
@@ -59,9 +59,11 @@ def get_pdf
     end
 
     def submit_combined_fsr(form)
-      persist_form_submission(form)
+      form_submission = persist_form_submission(form)
       vba_status = submit_vba_fsr(form) if selected_vba_debts(form['selectedDebtsAndCopays']).present?
-      vha_status = submit_vha_fsr(form) if selected_vha_copays(form['selectedDebtsAndCopays']).present?
+      if selected_vha_copays(form['selectedDebtsAndCopays']).present?
```

Comment:
I would stick with single line statements.

Reply:
@RachalCassity Agreed it looks cleaner, rubocop auto formatted here. Just updated the variable name to avoid breaking line length rules